docs/tips/03-if-else-in-JSX.md

```diff
@@ -30,7 +30,7 @@ React.createElement("div", {id: if (condition) { 'msg' }}, "Hello World!");
 That's not valid JS. You probably want to make use of a ternary expression:
 
 ```js
-ReactDOM.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
+ReactDOM.render(<div id={condition ? 'msg' : null}>Hello World!</div>, mountNode);
```

Comment:
This change is fine but is unrelated to the rest of your description. This is for the `id` attribute and will never render to `<span>`